scripts/pkg/install.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/sh
```

Comment:
May I go ahead an replace this with a `#!/usr/bin/env bash` ?
This script uses bash features such as `[[ ... ]]` and users are instructed to run it with bash.

Reply:
I think it's okay given the fact that our installation method directly suggests using `bash`. Do you see any objections to that @medikoo ?

Reply:
Yes, I think it's a right call. Good suggestion @alegonz